src/components/nodes/endEvent/index.js

```diff
@@ -25,11 +25,6 @@ export default {
       }),
     });
   },
-  /**
-   * Validate whether to accept an outgoing flow to the node
-   *
-   * @param node
-   */
   validateOutgoing() {
```

Comment:
Is there any way this method name can be changed to match the comment? `allowOutgoingFlow()` would be a bette name IMO.

Reply:
I might have to change unrelated code in sequence and message flow.